### DISCUSSION TITLE
Customize footer checkbox text

### DISCUSSION THREAD

**Discussion by Atches** [renovatebot/renovate#20046] [Suggest an Idea]
### What would you like Renovate to be able to do?

We would like the footer checkbox text to be customizable.

### If you have any ideas on how this should be implemented, please tell us here.

We should add to the configuration Renovate receives, a new property `prFooterMessage` that will contain the text to use.

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
PR footer text is already customizable. See https://docs.renovatebot.com/configuration-options/#prfooter
